pyproject.toml

```diff
@@ -5,7 +5,28 @@ description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.13"
 dependencies = [
-    "dagster>=1.11.2",
+    "dagster>=1.11.3",
+    "dagster-dg-cli>=1.11.3",
+    "dagster-postgres>=0.27.3",
+    "dagster-webserver>=1.11.3",
```

Comment:
I needed to add some additional dagster dependencies to the environment to get it to run. Bumped the minimum version to 1.11.3 since that's the first version with robust Python 3.13 supportl